packages/progress-bar/src/Kaizen/ProgressBar.elm

```diff
@@ -0,0 +1,96 @@
+module Kaizen.ProgressBar exposing (Mood(..), view)
+
+import Box.Box as Box exposing (..)
+import CssModules exposing (css)
+import Heading.Heading as Heading exposing (TypeVariant(..))
+import Html exposing (..)
+import Html.Attributes exposing (attribute, style)
+
+
+type alias Configuration =
+    { progressPercentage : Int
+    , mood : Mood
+    }
```

Comment:
I gather you're deliberately preferring the simplicity of an plain record rather than an opaque `Configuration` type with exposed constructors. I guess the question is, will this lead to more breaking changes in the API as we add features to this component in the future?

Not strongly opinionated on this, but I think I'd have gone with the opaque type myself, knowing that there are already features designed for this component that aren't implemented here.

Reply:
TBH this is just what I copied from murmur and I didn't think of it. I agree, opaque types are generally better. I've pushed a change.